README.md

```diff
@@ -42,8 +42,8 @@ fs.src(['*.js', '!b*.js'])
     - Default is `process.cwd()`.
 
   - base - Specify the folder relative to the cwd. This is used to determine the file names when saving in `.dest()`.
-    - Default is where the glob begins if any.
-    - Default is `process.cwd()` if there is no glob.
+    - Default is where the glob begins, if any. For example, `path/to/**/*.js` would resolve to `path/to`.
```

Comment:
Thought I'd add an example, especially since the second point to this behaves differently and also has an example.